### ISSUE TITLE
Display beat/bassline notes on the clip and store note data inside the clip, not in the whole track

### ISSUE THREAD

**Issue by MWstudios** [LMMS/lmms#6656]
### Enhancement Summary
Beat/bassline patterns would be displayed on clips like piano roll clips do. Also, the notes would be stored inside the clip, which means multiple clips with different note patterns could exist on a single track.
#### Justification
This is a feature which I've wanted for years. When you use a specific pattern only once per song, you need to make a separate track just for that single clip alone, leaving a lot of unnecessary empty space in the main editor when this happens multiple times.

**Comment by musikBear** [comment]
> When you use a specific pattern only once per song, you need to make a separate track just for that single clip alone

You do not have to.
A BB-track does not have to contain elements from all instruments in the BB-editor.
You can simply make BB-tracks of single instruments, and variations.
That is what i call the LEGO model.
I explained the various BB-editor strategies in our old wiki, but that is nomore.
Most is also in this video
https://www.youtube.com/watch?v=Gh6cgBIx77U&list=PLMvza7ri1L0eduljUjLSsXRvIJ8rD8Ohs&index=4&

**Comment by musikBear** [comment]
> I see no reason why the same thing couldn't apply to BB-tracks.

They are fundamental different.

**Comment by qnebra** [comment]
You just described 'standard' clips from Song Editor itself, which does majority of your request feature. You want individually stored patterns, and each one can be unique? Use Song Editor clip, not from "Pattern Editor".

**Comment by MWstudios** [comment]
I'm aware that a Song Editor track stores separate note data per clip. However, that only applies to only one instrument, meanwhile the BB track can group multiple instruments together and allows easier repetition, so I thought would be beneficial if those clips could store their own patterns.

**Comment by allejok96** [comment]
This could also be solved with some kind of layering/grouping in song editor I guess.

Closing as a duplicate of #454 (and 4877), but thank you for the good example use case.
